modules/cnf-creating-nrop-cr-hosted-control-plane.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-creating-nrop-cr-hosted-control-plane_{context}"]
+= Creating the NUMAResourcesOperator custom resource for the hosted control plane
+
+When you have installed the NUMA Resources Operator, then create the `NUMAResourcesOperator` custom resource (CR) that instructs the NUMA Resources Operator to install all the cluster infrastructure needed to support the NUMA-aware scheduler on hosted control plane, including daemon sets and APIs.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the NUMA Resources Operator.
+
+.Procedure
+
+.  Export the management cluster kubeconfig:
```

Comment:
```suggestion
.  Export the management cluster `kubeconfig` file by running the following command:
```
OCP doc guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command